docs/tutorials/notes-mvvm/includes/mvvm.md

```diff
@@ -0,0 +1,36 @@
+---
+author: adegeo
+ms.author: adegeo
+ms.date: 01/05/2023
+ms.topic: include
+---
+
+The .NET MAUI developer experience typically involves creating a user interface in XAML, and then adding code-behind that operates on the user interface. Complex maintenance issues can arise as apps are modified and grow in size and scope. These issues include the tight coupling between the UI controls and the business logic, which increases the cost of making UI modifications, and the difficulty of unit testing such code.
+
+The MVVM pattern helps cleanly separate an app's business and presentation logic from its user interface (UI). Maintaining a clean separation between app logic and the UI helps address numerous development issues and makes an app easier to test, maintain, and evolve. It can also significantly improve code reuse opportunities and allows developers and UI designers to collaborate more easily when developing their respective parts of an app.
```

Comment:
Expand MVVM here? Seems to be the first usage in this tutorial (previously was in model.md).